src/main/scala/com/danielasfregola/twitter4s/http/serializers/JsonSupport.scala

```diff
@@ -8,7 +8,7 @@ import org.json4s.{DefaultFormats, Formats, native}
 
 trait JsonSupport extends Json4sSupport {
 
-  implicit val serialization = native.Serialization
```

Comment:
Any particular reason why this has been changed?

Reply:
Due to this [bug](https://github.com/hseeberger/akka-http-json/issues/148), it unintendedly convert into `application/json` where it is not supposed to. The suggestion I got from that issue is to limit the scope of `JsonSupport` into the place where it is needed. But, that would lead to a bigger change.

Instead I limit the scope of serialization, since [Json4sSupport](https://github.com/hseeberger/akka-http-json/blob/master/akka-http-json4s/src/main/scala/de/heikoseeberger/akkahttpjson4s/Json4sSupport.scala) needs an implicit `Serialization`, so it will not unmarshall where its not needed. This leads to a smaller change and can be easily reverted.

I'll try to send a PR to revert this once the bug is fixed.

Reply:
Thanks for clarifying! It makes sense 👍